README.md

```diff
@@ -15,42 +15,25 @@ Package: https://hex.pm/packages/exqlite
 
 ## Caveats
 
-* Prepared statements are not cached.
 * Prepared statements are not immutable. You must be careful when manipulating
   statements and binding values to statements. Do not try to manipulate the
   statements concurrently. Keep it isolated to one process.
-* Simultaneous writing is not supported by SQLite3 and will not be supported
-  here.
-* All native calls are run through the Dirty NIF scheduler.
-* Datetimes are stored without offsets. This is due to how SQLite3 handles date
-  and times. If you would like to store a timezone, you will need to create a
-  second column somewhere storing the timezone name and shifting it when you
-  get it from the database. This is more reliable than storing the offset as
-  `+03:00` as it does not respect daylight savings time.
+* All native calls are run through the Dirty IO NIF scheduler.
 * When storing `BLOB` values, you have to use `{:blob, the_binary}`, otherwise
-  it will be interpreted as a string. 
+  it will be interpreted as a string.
 
 ## Installation
 
 ```elixir
 defp deps do
   [
+    # TODO
     {:exqlite, "~> 0.17"}
```

Comment:
- [ ] Version would need to be bumped.

Reply:
Also I'm getting this error when compiling

```
==> exqlite
Error happened while installing exqlite from precompiled binary: "missing checksum.exs file".

Attempting to compile exqlite from source...
 CC sqlite3_nif.o
 CC sqlite3.o
 LD sqlite3_nif.so
```

Not sure if it's caused by my changes. I'll investigate.

Reply:
For this major move I'd bump major to `1.0` since it is a breaking change.